lib/schedule/piece.ex

```diff
@@ -74,8 +74,8 @@ defmodule Schedule.Piece do
     Enum.all?(piece.trips, &match?(%Schedule.Trip{service_id: nil}, &1))
   end
 
-  @spec is_for_now?(t(), Util.Time.time_of_day()) :: boolean()
-  def is_for_now?(piece, now_time_of_day) do
+  @spec for_now?(t(), Util.Time.time_of_day()) :: boolean()
+  def for_now?(piece, now_time_of_day) do
     piece.start_time <= now_time_of_day and piece.end_time > now_time_of_day
```

Comment:
I kind of like `current` rather than `active`, since these represent _scheduled_ service that may or may not actually be active in the sense of having a vehicle running them.

Reply:
Just renamed it to `current?`